### PR TITLE
[15.0][MIG] sale_timesheet_order_line_sync: Migration to version 15.0

### PR DESCRIPTION
**Pull request by stefan-tecnativa**
CC @Tecnativa TT38403

The functionality of this module has been added to Odoo core.

@carlosdauden @pedrobaeza

### CODE REVIEW

**Review by carlosdauden — APPROVED** [review]

**Review by chienandalu — APPROVED** [review]

File: openupgrade_scripts/apriori.py [hunk 1]
```diff
@@ -49,6 +49,8 @@
     # OCA/project
     "project_mail_chatter": "project",
     "project_task_dependency": "project",
+    # OCA/timesheet
+    "sale_timesheet_order_line_sync": "sale_project",
```
Comment (MiquelRForgeFlow) [inline openupgrade_scripts/apriori.py]:
```suggestion
    "sale_timesheet_order_line_sync": "sale_timesheet",
```

**Review by MiquelRForgeFlow — DISMISSED** [review]

File: openupgrade_scripts/apriori.py (hunk 1 above)
Comment (pedrobaeza) [reply openupgrade_scripts/apriori.py]: Not really. The one that fits more is this one.

Comment (MiquelRForgeFlow) [reply openupgrade_scripts/apriori.py]: Firstly, `sale_timesheet_order_line_sync` depends on `sale_timesheet`. And secondly, it's code is simple:

Selection_713

and by looking similar things in the code of odoo 🔎:
- sale_project: nothing similar
- sale_project_account: nothing similar
- sale_timesheet:

Selection_714
Selection_716
Selection_715

Oh, there are several similarities in the code of sale_timesheet, match?  🎉

BTW, If you think I am wrong, I think you should give your arguments about "fitting" 🤔

Comment (pedrobaeza) [reply openupgrade_scripts/apriori.py]: OK, it doesn't matter anyway. Put again you request changes.

Comment (MiquelRForgeFlow) [inline openupgrade_scripts/apriori.py]:
```suggestion
    "sale_timesheet_order_line_sync": "sale_timesheet",
```
As discussed above :)

**Review by MiquelRForgeFlow — CHANGES_REQUESTED** [review]

File: openupgrade_scripts/apriori.py (hunk 1 above)
Comment (stefan-tecnativa) [reply openupgrade_scripts/apriori.py]: Done

**Comment by pedrobaeza** [comment]
Comment fixed:

/ocabot merge nobump

**Pull request merged**